flagging_site/templates/api/index.html

```diff
@@ -26,4 +27,52 @@ <h2>Available Endpoints</h2>
         <li><a href="{{ url_for('api.reachesapi') }}" target="_blank">Flagging Model Outputs</a></li>
         <li><a href="{{ url_for('api.boathousesapi') }}" target="_blank">Boathouses</a></li>
     </ul>
+    <h2>Sample code to call API</h2>
+    <ul>
+        <li>Calling Flagging Model Outputs</li>
+        <ul>
+            <li>
+                using Python
+                <pre class="prettyprint">
+                    import pandas as pd   
+                    url = "http://127.0.0.1:5000/api/v1/model"
+                    model_output = pd.read_json(url)
+                    print(model_output.head())
+                </pre>
+            </li>
+            <li>
+                using R
+                <pre class="prettyprint">
```

Comment:
Try `<pre class="prettyprint lang-python">` for the Python code and `<pre class="prettyprint lang-r">` for the R code.